checkstyle/src/main.rs

```diff
@@ -0,0 +1,124 @@
+use regex::Regex;
+use std::fs::{self};
+
+fn get_file(filename: &str) -> String {
+    fs::read_to_string(filename).expect("Failed to read file")
+}
```

Comment:
Several things here:

1) `get_file` implies that the result of the function would be a file, i.e. some representation of it. `read_file_contents` would be a better name.
2) `filename` is very limiting. Presumably, we want this function to accept a path to any file, not just an entry in our cwd, so the better name would be `file_path`.
3) Speaking of which, if this is a path, why is this a bare `&str` rather than a more explicit `&Path`?